Formula/perl.rb

```diff
@@ -30,6 +30,11 @@ class Perl < Formula
   skip_clean "lib/perl5/site_perl"
 
   def install
+    # Apply upstream commit to remove nsl from libswanted:
+    # https://github.com/Perl/perl5/commit/7e19816aa8661ce0e984742e2df11dd20dcdff18
+    # Remove with next tagged release that includes the change.
+    inreplace "Configure", " nsl ", ""
```

Comment:
Can we use a patch instead of an `inreplace`?

Reply:
Using this commit should probably work:

https://github.com/Perl/perl5/commit/7e19816aa8661ce0e984742e2df11dd20dcdff18.patch?full_index=1

Reply:
Okay I'll try to apply that commit and switch to a patch if the commit cannot be applied cleanly.  We are at 5.34 right now and `blead` is at 5.37 so I'm not sure if the commit will apply.  Nonetheless it's definitely worth trying that first.

Reply:
The commit should work based on local testing.